lldb/include/lldb/Core/ModuleList.h

```diff
@@ -285,7 +285,7 @@ class ModuleList {
   /// \see Module::FindFunctions ()
   void FindFunctions(ConstString name, lldb::FunctionNameType name_type_mask,
                      const ModuleFunctionSearchOptions &options,
-                     SymbolContextList &sc_list) const;
+                     const SymbolContext &sc, SymbolContextList &sc_list) const;
```

Comment:
We might be able to reduce the number of changes in the codebase if we add the symbol context at the end and make it a pointer. If the pointer is NULL, then no context matching will occur, else it will try to use the `sc->module_sp`. It will help reduce the changes in our codebase and having a pointer might help indicate that the symbol context is optional.

Maybe we do:
```

void FindFunctions(ConstString name, lldb::FunctionNameType name_type_mask,
                     const ModuleFunctionSearchOptions &options,
                     SymbolContextList &sc_list, const SymbolContext *sc = nullptr) const;